_data/pull-db.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+if [[ $# -ge 1 ]] ; then
+    dbPath=$1
+else
+    mkdir -p ./_data/
+fi
+
+BUCKET_NAME="hypelabs-public/crocswap"
```

Comment:
lets move these into .env, or a global config, we're defining them here and in .go codebase